recipes/scicookie/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: e17a7641a8c0ed6bc27e7826a0db45db4d6aa9dfc5e939ca7718df6ffb5030ac
+
+build:
+  noarch: python
```

Comment:
```suggestion
  skip: true  # [win]
```

Reply:
No need. This still can be noarch but not available on Windows due to a "solve error."

Reply:
ok .. got it! thanks for the explanation!